warehouse/accounts/security_policy.py

```diff
@@ -332,22 +332,29 @@ def _check_for_mfa(request, context) -> WarehouseDenied | None:
     ]
 
     if (
-        request.matched_route.name == "forklift.legacy.file_upload"
-        or request.matched_route.name.startswith("manage")
-        and request.matched_route.name != "manage.account"
-        and not any(
-            request.matched_route.name.startswith(route) for route in _exempt_routes
-        )
-        and not request.identity.has_two_factor
-    ) and (
-        # Start enforcement from 2023-08-08, but we should remove this check
-        # at the end of 2023.
```

Comment:
We lose this comment ass a result - intended?

Reply:
nope! I'm just messy.

Reply:
8e45163457fd8c6e91ac35b2481100f175ba5981